src/puripuly_heart/ui/controller.py

```
@@ -886,15 +890,33 @@ def _managed_openrouter_local_key_available(self) -> bool:
+        has_key = resolution.api_key is not None
+        logger.info("[ManagedAuth] _managed_openrouter_local_key_available: non-China mode, has_key=%s", has_key)
+        return has_key
+
+    def _managed_qq_key_available(self) -> bool:
+        if self.settings is None:
+            return False
+        try:
+            secrets = create_secret_store(self.settings.secrets, config_path=self.config_path)
+            result = bool(secrets.get("openrouter_managed_qq_api_key"))
+            logger.info("[QQAuth] _managed_qq_key_available: %s", result)
+            return result
```

Ruff (0.15.15)

[warning] Consider moving this statement to an `else` block

(TRY300)